src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -579,7 +583,8 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
             // Get entire row
             String tableRow =
                     FeedbackQuestionFormTemplates.populateTemplate(tableBodyTemplate,
-                            "${subQuestion}", StringHelper.integerToLowerCaseAlphabeticalIndex(j + 1) + ") " + Sanitizer.sanitizeForHtml(rubricSubQuestions.get(j)),
+                            "${subQuestion}", StringHelper.integerToLowerCaseAlphabeticalIndex(j + 1) + ") "
+                                                    + Sanitizer.sanitizeForHtml(rubricSubQuestions.get(j)),
```

Comment:
Same as above:
Maybe instead of indenting the `+` 8 spaces from the `,`, we indent 8 spaces from the `[S]tringHelper`, which is where the thing begins